src/dswx_sar/save_mgrs_tiles.py

```diff
@@ -649,10 +650,27 @@ def run(cfg):
                 no_data=no_data_raster)
 
     # Get list of MGRS tiles overlapped with mosaic RTC image
+    mgrs_meta_dict = {}
+
     if database_bool:
-        mgrs_tile_list = get_intersecting_mgrs_tiles_list_from_db(
+        mgrs_tile_list, most_overlapped = get_intersecting_mgrs_tiles_list_from_db(
```

Comment:
Cannot post the comment directly to `get_intersecting_mgrs_tiles_list_from_db()`, but it seems `most_overlapped` is not mentioned in the docstring. Suggest revising the docstring.